src/components/modals/Agenda/Agenda.tsx

```diff
@@ -44,11 +49,17 @@ export default function Agenda({
                 sx={{
                     display: "flex",
                     alignItems: "center",
+                    justifyContent: "space-between",
                 }}
             >
                 <Typography variant="h6" component="h2">
                     Min timeplan
                 </Typography>
+                <Tooltip title={chainProfile.name}>
+                    <Avatar sx={{ width: 22, height: 22 }} src={chainProfile.images.common.smallLogo ?? ""}>
+                        {chainProfile.identifier}
+                    </Avatar>
+                </Tooltip>
```

Comment:
Is the extra logo really necessary? Do you feel that it's unclear what chain you have selected?

Reply:
Well, now that I think about it, it is only really relevant to me, since I bounce around a lot 😝

Reply:
Fixed in https://github.com/mathiazom/rezervo-web/pull/100/commits/d73728fb6a180ecda201f7a287c8fa16ee6e9e99